src/main/java/net/masterthought/cucumber/reducers/ReducingMethod.java

```diff
@@ -13,6 +13,34 @@
      */
     MERGE_FEATURES_BY_ID,
 
+    /**
+     * Merge features and scenarios from different JSON files of different runs
+     * into a single report by features' and scenarios' ids.
+     *
+     * Merging rules:
+     * - Every new feature which is not in the result list is appended to the end.
+     *
+     * - When the results list already has a feature with such Id then we go down and apply the rules below to the scenarios:
+     *
+     *      1. if there is no scenario with a given Id in the feature's elements list
+     *         then add the scenario to the end of the list.
+     *
+     *      2. if scenario has a background (which is a previous element in the elements list)
```

Comment:
scenario which is in the retested feature or the one which has been already added to merged report?